dev-python/tikzplotlib/tikzplotlib-0.9.4.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+
+inherit distutils-r1 virtualx
+
+DESCRIPTION="Convert matplotlib figures into TikZ/PGFPlots"
+HOMEPAGE="https://github.com/nschloe/tikzplotlib"
+SRC_URI="https://github.com/nschloe/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	app-text/texlive
```

Comment:
According to the conversation only the test phase has failed. Maybe make it depend on 'test'?

Reply:
The package is useless without texlive, the whole point of the thing is to convert python matplotlib's plots to texlive's pgfplots, this is why I added it to RDEPEND.